test/python/compiler/test_transpiler.py

```diff
@@ -1362,7 +1362,7 @@ def test_transpile_optional_registers(self, optimization_level):
         out = transpile(qc, FakeBoeblingen(), optimization_level=optimization_level)
 
         self.assertEqual(len(out.qubits), FakeBoeblingen().configuration().num_qubits)
-        self.assertEqual(out.clbits, clbits)
+        self.assertEqual(len(out.clbits), len(clbits))
```

Comment:
how come?

Reply:
An equality check here is checking its the same object as the input. Prior to minimum point this would be true because the clbit is passed by reference through the dag to the output circuit. But with minimum point the output dagcircuit is deep copied meaning it's a different memory address/object than the original circuit's and this check would no longer be true.

Reply:
Yeah, the fact that they were passed through perfectly before is more just a side effect of how little we touch `Clbit` instances in transpilation passes, rather than something we ought to be asserting as an invariant of transpile.